### ISSUE TITLE
[pi-ui] Use exposed classNames instead of external npm package

### ISSUE THREAD

**Issue by amassarwi** [decred/decrediton#2513]
Currently we are using a lib with the name `classnames`, pi-ui exposes a very efficient function with the same purpose - `import { classNames } from "pi-ui"` that we should use instead.

Also, we have many places where we structure a className using string manipulation we should replace all of these with `classNames(class1 , conditioned class2, ...)`.

**Comment by amassarwi** [comment]
@alexlyp
Should be straight forward change:
We have 10 occurrences of `import cx from "classnames";` which should be replace with `import { classNames } from "pi-ui";` - I will already changed couple on #2510.

But also we have stuff like:
```
 const className = [
      "trezor-passphrase-modal",
      this.props.isGetStarted ? "get-started" : ""
    ].join(" ");
```

which can be replaced with: `classNames("trezor-passphrase-modal", this.props.isGetStarted && "get-started")` - I already refactored dozens while migrating `.less` files to css modules.

Actually when I think about now this can be done in two steps first ditch external lib (used only 10 times) and the second part can be done incrementally while migrating to css modules.

What do you think? I can open a PR for the first part.

**Comment by alexlyp** [comment]
Yah makes sense to do the first part.

**Comment by amassarwi** [comment]
This issue can be closed with #2514, I added the second part to the css modules issue, as it can be done step by step while migrating to css modules.
